metricbeat/module/mongodb/dbstats/data.go

```diff
@@ -6,27 +6,27 @@ import (
 )
 
 var schema = s.Schema{
-	"db":           c.Str("db"),
-	"collections":  c.Int("collections"),
-	"objects":      c.Int("objects"),
-	"avg_obj_size": c.Int("avgObjSize"),
-	"data_size":    c.Int("dataSize"),
-	"storage_size": c.Int("storageSize"),
-	"num_extents":  c.Int("numExtents"),
-	"indexes":      c.Int("indexes"),
-	"index_size":   c.Int("indexSize"),
+	"db":                 c.Str("db"),
+	"collections":        c.Int("collections"),
+	"objects":            c.Int("objects"),
+	"avg_obj_size.bytes": c.Int("avgObjSize"),
```

Comment:
This will make it incompatible with elasticsearch 2.x. You must use `s.Object{ ... } ` to add the nesting.